jupyterlab_nodeeditor/yggdrasil_support.py

```diff
@@ -8,30 +8,53 @@
 import yggdrasil.yamlfile
 import re
 
-photosynthesis_model = {}
 with open(ex_yamls['fakeplant']['python'], "r") as test_model:
     photosynthesis_model = yaml.safe_load(test_model)['model']
     
-# Hard coded function to clean up Photosynthesis model
+# Helper function to make a base empty dictionary
+# May not be needed in the future, but will cut time down for now
+
+def new_yggjlne_dict(num_inputs = 1, num_outputs = 1, name = "New Model"):
```

Comment:
I don't think we need this, do we?  It's initializing a set of dictionaries and empties and whatnot, but can't we just do this inline in the function where we do the iteration?